CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  * Fix issue with `with` failing with a traceback error when an exception is thrown (#714).
  * Fix issue with `sort-*` family of funtions returning an error on an empty seq (#716).
  * Fix issue with `intern` failing when used (#725).
+ * Fix issue with `ns` not being available after `in-ns` on the REPL (#xxx).
```

Comment:
```suggestion
 * Fix issue with `ns` not being available after `in-ns` on the REPL (#727).
```

Reply:
oops, sorry I have committed before seeing the comment, I've put the reported issue num #718 instead of PR if this is ok with you?

Reply:
I am ok with either -- thanks!